src/CMakeLists.txt

```diff
@@ -190,57 +202,90 @@ target_include_directories(libsouffle
 # install souffle directory
 install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/include/souffle DESTINATION include)
 
+add_library(compiled STATIC dummy.cpp)
```

Comment:
The `compiled` target is a dummy library that only purpose is to gather all compilation options and properties generated by CMake, that will then be injected in `souffle-compile.py`.